Networking/Networking/Model/Refund/LineItemRefund.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// Represents a Line Item to be Refunded
+///
+public struct LineItemRefund {
+
+    ///  ID from the order's line_items
+    ///
+    public let itemID: String
+
+    /// Item quantity
+    ///
+    public let quantity: Int
+
+    /// Item price without tax
+    ///
+    public let refundTotal: String?
+
+    /// Taxes to refund
+    public let refundTax: [TaxRefund]?
+
+    /// Line Item Refund struct initializer
+    ///
+    public init(itemID: String, quantity: Int, refundTotal: String?, refundTax: [TaxRefund]?) {
+        self.itemID = itemID
+        self.quantity = quantity
+        self.refundTotal = refundTotal
+        self.refundTax = refundTax
+    }
+
+    public func toDictionary() -> [String: Any] {
+        var dict: [String: Any] = ["qty": quantity]
```

Comment:
I would suggest declaring the keys in a private enumeration, or struct, to avoid using strings directly

Reply:
You are totally right! I'm changing it 👍